### PR TITLE
Fixed description commands

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

While I was practicing my presentation I have noticed that some atlascli commands still have mongocli in their descriptions

### CODE REVIEW

File: internal/cli/atlas/dbusers/dbusers.go [hunk 1]
```diff
@@ -17,11 +17,21 @@ package dbusers
...
+	exampleCmd = "mongocli atlas"
+)
+
 func Builder() *cobra.Command {
 	const use = "dbusers"
+
+	if config.ToolName == config.AtlasCLI {
+		exampleCmd = "atlas"
+	}
+
```
Comment (blva) [inline internal/cli/atlas/dbusers/dbusers.go:34]: [q] if we're adding config dependency anyways and using this in several places, should we have a method there to fetch this command?

**Review by blva — APPROVED** [review]
LGTM - one non-blocking question

File: internal/cli/atlas/dbusers/dbusers.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/atlas/dbusers/dbusers.go:34]: I don't have a strong opinion but I think config shouldn’t contain a constant used only for the description of commands as it is not configuration-related. I will leave this comment open to see if someone has a strong opinion on this.

**Comment by andreaangiolillo** [comment]
I am not waiting for doc review as I am only changing wrong examples

**Pull request merged**